netdata.spec.in

```diff
@@ -10,7 +10,7 @@
 %bcond_without  systemd  # systemd
 %bcond_with     nfacct   # build with nfacct plugin
 
-%if 0%{?fedora} || 0%{?rhel} >= 7
+%if 0%{?fedora} || 0%{?rhel} >= 7 || 0%{?suse_version} >= 1140
```

Comment:
this was invalid at first... we should avoid doing undefine and move the upper definition into proper conditionals.